Documentation/cloud-config.md

```diff
@@ -56,7 +56,7 @@ CoreOS tries to conform to each platform's native method to provide user data. E
 
 ### coreos
 
-#### etcd (deprecated)
+#### etcd (deprecated. see etcd2)
```

Comment:
Is the reversion of the docs change from the previous commit intentional?

Reply:
Heh, no. I'm not sure how I messed up the commit.